app/assets/stylesheets/global.css.scss

```diff
@@ -1,5 +1,5 @@
 #gradient {
-  background: red; /* For browsers that do not support gradients */
+  background:  #007EEE; /* For browsers that do not support gradients */
```

Comment:
Color literals like `#007EEE` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Use `//` comments everywhere<br>Color `#007EEE` should be written as `#007eee`<br>Colon after property should be followed by one space